target/linux/ath79/dts/qca9531_dlink_dch-g020-a1.dts

```diff
@@ -0,0 +1,174 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "dlink,dch-g020-a1", "qca,qca9531";
+	model = "D-Link DCH-G020 A1";
+
+	aliases {
+		label-mac-device = &eth0;
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	i2c {
+		compatible = "i2c-gpio";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		sda-gpios = <&gpio 0 GPIO_ACTIVE_HIGH>;
+		scl-gpios = <&gpio 1 GPIO_ACTIVE_HIGH>;
+
+		pca9554: pca9554@38 {
+			compatible = "nxp,pca9554";
+			reg = <0x38 0x1>;
+			#clock-cells = <0>;
+			gpio-controller;
+			#gpio-cells = <2>;
+		};
+
+		rtc: pt7c43390@30 {
+			compatible = "pericom,pt7c43390";
+			reg = <0x30 0x8>;
+		};
+	};
+
+	gpio-export {
+		compatible = "gpio-export";
+
+		usb_enable {
+			label = "pca9554:usb_enable";
```

Comment:
This is typically set up with three parts: "a:b:c"

Just have a look at what other devices do. Here, it might be something like "dch-g020-a1:power:usb"...

Reply:
Thanks, changing to `dlink:power:usb`.